jobs/JGDAS_GLOBAL_OCEAN_ANALYSIS_PREP

```diff
@@ -35,7 +35,8 @@ RUN=${GDUMP} YMD=${gPDY} HH=${gcyc} generate_com -rx \
 
 # Add UFSDA to PYTHONPATH
 ufsdaPATH="${HOMEgfs}/sorc/gdas.cd/ush/"
-PYTHONPATH="${PYTHONPATH:+${PYTHONPATH}:}${ufsdaPATH}"
+pyiodaPATH="${HOMEgfs}/sorc/gdas.cd/build/lib/python3.7/"
```

Comment:
we need to be careful about prescribing a python version here I think... but this is probably the best solution for now, sadly

Reply:
Yeah that's why I included you. I wasn't sure if we could do the same thing here as in GDASApp.

Reply:
I had the same concerns. @aerorahul?